### PR TITLE
Added SliderWrapper and find_window(s) aliases

### PR DESCRIPTION
**Pull request by cetygamer**

### CODE REVIEW

File: pywinauto/controls/uia_controls.py [hunk 1]
```diff
@@ -410,3 +413,75 @@ def select(self, start = 0, end = None):
...
+        return self.iface_RangeValue.CurrentLargeChange
+
+    #-----------------------------------------------------------
+    def value(self):
+        """Get current position of slider's thumb"""
+        return self.iface_RangeValue.CurrentValue
+
+    #-----------------------------------------------------------
+    def set_value(self, value):
+        """Set position of slider's thump"""
```
Comment (vasily-v-ryabov) [inline pywinauto/controls/uia_controls.py]: typo: `thump` -> `thumb`

File: pywinauto/controls/uia_controls.py [hunk 2]
```diff
@@ -410,3 +413,75 @@ def select(self, start = 0, end = None):
...
+        elif isinstance(value, six.integer_types):
+            value_to_set = value
+        elif isinstance(value, six.text_type):
+            value_to_set = float(value)
+        else:
+            raise ValueError("value should be either string or number")
+
+        min_value = self.min_value()
+        max_value = self.max_value()
+        if value_to_set < min_value or value_to_set > max_value:
```
Comment (vasily-v-ryabov) [inline pywinauto/controls/uia_controls.py]: Can be simplified as `not (min_value <= value_to_set <= max_value)` (QuantifiedCode warning).

File: pywinauto/unittests/test_UIAWrapper.py [hunk 3]
```diff
@@ -482,6 +482,66 @@ def testSelect(self):
...
+
+            # start the application
+            app = Application(backend='uia')
+            app = app.start(wpf_app_1)
+
+            self.app = app
+            self.dlg = app.WPFSampleApplication
+
+            from pywinauto.controls.uia_controls import SliderWrapper
+            self.slider = SliderWrapper(self.dlg.Slider.element_info)
```
Comment (airelil) [inline pywinauto/unittests/test_UIAWrapper.py]: I'd prefer: ```self.dlg.Slider.WrapperObject()```, so we test that the Slider can be 'auto-detected'

File: pywinauto/controls/uia_controls.py [hunk 4]
```diff
@@ -410,3 +413,75 @@ def select(self, start = 0, end = None):
...
+        IUIA().UIA_dll.UIA_SliderControlTypeId,
+    ]
+    has_title = False
+
+    #-----------------------------------------------------------
+    def __init__(self, elem_or_handle):
+        """Initialize the control"""
+        super(SliderWrapper, self).__init__(elem_or_handle)
+        # Get interface to work with this slider
+        self.iface_RangeValue = uia_defs.get_elem_interface(self.element_info.element, "RangeValue")
```
Comment (airelil) [inline pywinauto/controls/uia_controls.py]: Nice. Maybe we should precache ```self.iface``` in other wrappers as well

File: pywinauto/controls/uia_controls.py [hunk 5]
```diff
@@ -410,3 +413,75 @@ def select(self, start = 0, end = None):
...
+    def set_value(self, value):
+        """Set position of slider's thump"""
+        if isinstance(value, float):
+            value_to_set = value
+        elif isinstance(value, six.integer_types):
+            value_to_set = value
+        elif isinstance(value, six.text_type):
+            value_to_set = float(value)
+        else:
+            raise ValueError("value should be either string or number")
```
Comment (airelil) [inline pywinauto/controls/uia_controls.py]: There is not test for that

File: pywinauto/findwindows.py [hunk 6]
```diff
@@ -271,6 +283,18 @@ def _title_match(w):
...
+    """
+    Find elements based on criteria passed in and return list of their handles
+
+    Calls find_elements with exactly the same arguments as it is called with
+    so please see find_elements for a description of them.
+    """
+    kwargs['backend'] = 'native'
+    elements = find_elements(**kwargs)
+    return [elem.handle for elem in elements]
+
```
Comment (airelil) [inline pywinauto/findwindows.py]: There are not tests for find_windows and find_window

File: pywinauto/controls/uia_controls.py [hunk 7]
```diff
@@ -229,8 +229,10 @@ def item_count(self):
 class EditWrapper(UIAWrapper.UIAWrapper):
 
     """Wrap an UIA-compatible Edit control"""
+    # TODO: this class supports only 1-line textboxes so there is no point
+    # TODO: in methods such as line_count(), line_length(), get_line(), etc
 
-    controltypes = [
+    control_types = [
```
Comment (airelil) [inline pywinauto/controls/uia_controls.py]: Looks as using 'WrapperObject' revealed a couple of bugs :-)

Comment (vasily-v-ryabov) [inline pywinauto/controls/uia_controls.py]: Hmm... Probably we need some warning in metaclass to detect such cases. Good catch, Alexander.

**Pull request merged**